### PR TITLE
Modified docs to match responses

### PR DESCRIPTION
**Pull request by MaureenMOguche**

### CODE REVIEW

File: eagles-food-backend/appsettings.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "prod db connection string"
+    "DefaultConnection": ""
```
Comment (poopsicles) [inline eagles-food-backend/appsettings.json:3]: hm does this matter? it still uses the development one anyway

File: eagles-food-backend/DOCUMENTATION.md [hunk 2]
```diff
@@ -526,6 +527,7 @@ Organizations are like banks, they hold users and allow them to transfer and red
       "role": "Admin"
     }
   ],
+  }
```
Comment (poopsicles) [inline eagles-food-backend/DOCUMENTATION.md:530]: oh there's also a "others" section thats just like org, but for people in the other organisations

it wont show up in your normal testing cos everyone is in the same org here, but just duplicate the org list here and add it under

File: eagles-food-backend/DOCUMENTATION.md [hunk 3]
```diff
@@ -167,7 +167,7 @@ Organizations are like banks, they hold users and allow them to transfer and red
       "ProfilePic": "",
       "Email": "john@doe.com",
       "Phone": "123456",
-      "IsAdmin": "False",
+      "IsAdmin": "True",
       "LunchCreditBalance": "",
```
Comment (poopsicles) [inline eagles-food-backend/DOCUMENTATION.md:171]: balance isnt 0 by default?

File: eagles-food-backend/DOCUMENTATION.md
Comment (poopsicles) [inline eagles-food-backend/DOCUMENTATION.md:1]: Users are now added to the "eagles food" org by default

**Review by poopsicles — CHANGES_REQUESTED** [review]

File: eagles-food-backend/appsettings.json (hunk 1 above)
Comment (MaureenMOguche) [reply eagles-food-backend/appsettings.json:3]: Okay. I'd return the test db connection string then

File: eagles-food-backend/DOCUMENTATION.md
Comment (MaureenMOguche) [reply eagles-food-backend/DOCUMENTATION.md:1]: Not sure I get this

**Review by poopsicles — APPROVED** [review]

**Pull request merged**